web/src/redux/apis/TraceTest.api.ts

```diff
@@ -48,6 +61,49 @@ const TraceTestAPI = createApi({
         {type: Tags.TEST, id: test?.id},
       ],
     }),
+    getEnvList: build.query<IEnvironment[], {take?: number; skip?: number; query?: string}>({
+      query: ({take = 25, skip = 0, query = ''}) => `/tests?take=${take}&skip=${skip}&query=${query}`,
+      providesTags: () => [{type: Tags.ENVIRONMENT, id: 'LIST'}],
+      transformResponse: () => [
+        {
```

Comment:
Should we use models for environment lists? instead of passing down the raw javascript objects

Reply:
sure: done! ✅